app/src/main/kotlin/com/simplemobiletools/notes/dialogs/ImportFolderDialog.kt

```diff
@@ -0,0 +1,72 @@
+package com.simplemobiletools.notes.dialogs
+
+import android.support.v7.app.AlertDialog
+import android.view.ViewGroup
+import com.simplemobiletools.commons.extensions.*
+import com.simplemobiletools.notes.R
+import com.simplemobiletools.notes.activities.SimpleActivity
+import com.simplemobiletools.notes.extensions.dbHelper
+import com.simplemobiletools.notes.helpers.TYPE_NOTE
+import com.simplemobiletools.notes.models.Note
+import kotlinx.android.synthetic.main.dialog_open_file.view.*
+import java.io.File
+
+class ImportFolderDialog(val activity: SimpleActivity, val path: String, val callback: (id: Int) -> Unit) : AlertDialog.Builder(activity) {
+    private var dialog: AlertDialog
+
+    init {
+        val view = (activity.layoutInflater.inflate(R.layout.dialog_import_folder, null) as ViewGroup).apply {
+            open_file_filename.text = activity.humanizePath(path)
+        }
+
+        dialog = AlertDialog.Builder(activity)
+                .setPositiveButton(R.string.ok, null)
+                .setNegativeButton(R.string.cancel, null)
+                .create().apply {
+                    activity.setupDialogStuff(view, this, R.string.import_folder) {
+                        getButton(AlertDialog.BUTTON_POSITIVE).setOnClickListener {
+                            val updateFilesOnEdit = view.open_file_type.checkedRadioButtonId == R.id.open_file_update_file
+                            saveFolder(updateFilesOnEdit)
+                        }
+                    }
+                }
+    }
+
+    private fun saveFolder(updateFilesOnEdit: Boolean) {
+        val folder = File(path)
+        var lastSavedNoteId = -1;
+        folder.listFiles({ file ->
+            val filename = file.path.getFilenameFromPath()
+            when {
```

Comment:
These checks were duplicated from MainActivity. Should I create some extension function for them?